209 configurable screens (#486)
SuperOfficeDocs/superoffice-docs: docs/ui/screen-designer/index.md

---
uid: screen-designer
title: Screen designer
description: Introduction to the Screen Designer in Settings and maintenance.
author: Bergfrid Dias
so.date: 10.10.2022
keywords: ui, screen, customization
so.topic: concept
so.version: 10
so.envir:
so.client:
---

# The Screen Designer

[!include[Requirement](../../../common/includes/req-dev-tools.md)]

The The **Screen Designer** in Settings and maintenance lets you design layouts that make the most sense in your organization. It replaces [PageBuilder][2] and the old way of customizing the UI - screen by screen.

The **Layouts** list displays any existing screen layouts (usually the default layout), with a preview screen on the right.

* Fields: Here you can find all available data fields for the selected screen layout. Use the filter function to find specific fields.
* Elements: Here you can add various non-data elements such as labels, dividers, links and, buttons.
* Settings: Customize appearance of individual field.

> * Columns: Choose how many columns (1-4) the fields and elements may occupy.

Reviewer: ```suggestion
* Columns: Lists column layout options to contain fields and elements.

```

* Tabs: Can be added, rearranged, and deleted.

## Where do I start?

> To open the Screen Designer, go to **Settings and maintenance** and select **Screen Designer** from the navigator.

Reviewer: Link to how/where to locate Settings and maintenance menu link.

Author: added

* Drag and drop field into the editor.
* Select a field to move it, or to update its settings.
* Available settings depend on the selected field type.

Learn more about [working with fields][1]

> [!NOTE]

> > After you publish a layout, users assigned to it must sign out of SuperOffice and back in to see the changes.

Reviewer: ```suggestion
> After a layout is published, user groups assigned to it must sign out of SuperOffice and back in to see the changes.

```

Author: It's not the group that must sign out. Thus going with the pattern, the text should be "user in groups assigned to it"

Reviewer: That's fine, however, stating "users assigned to it", what is 'it'? How about:

After a layout is published, users who are a member of the assigned user-groups must sign out of SuperOffice and back in to see the changes.

Author: "layout is published" is passive, 2nd clause is revised

### Undo changes

> If you need to revert to an earlier version of the screen, you can access the following options from the **Task** button:

Reviewer: ```suggestion
Use the **Task** button to revert to an earlier version of the screen.

```

> * **Reset to factory settings:** This option discards all changes and resets the screen layout to the standard layout delivered by SuperOffice.

Reviewer: ```suggestion
* **Reset to factory settings:** This option discards all changes and resets the screen layout to the default (system) layout delivered by SuperOffice.

```

* **Discard draft:** Discards all changes and resets to the published version of the screen layout.

## What if I don't use the Screen Designer?

> Without the Development Tools license, new cards take effects and the custom fields are displayed in two columns.

Reviewer: > Without the Development Tools license, new cards take effects and the custom fields are displayed in two columns.

What?

Author: revised
